src/schema.cpp

```diff
@@ -88,6 +88,19 @@ void Schema::validate() const
     }
 }
 
+namespace {
+struct IsNotRemoveProperty {
+    bool operator()(SchemaChange sc) const { return sc.visit(*this); }
```

Comment:
It's three pointers and an int, so with zero inlining it'd probably be faster as a reference but only very marginally so.